backend/src/main/java/dev/k2/deleteXsdFile/DeleteXsdFileController.java

```diff
@@ -0,0 +1,27 @@
+package dev.k2.deleteXsdFile;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.io.FileNotFoundException;
+
+@RestController
+@RequestMapping("api/v1/delete")
+@RequiredArgsConstructor
+@CrossOrigin("*")
+public class DeleteXsdFileController {
+
+    private final DeleteXsdFileService service;
+
+    @DeleteMapping("{fileId}")
+    public ResponseEntity<FileDeleteResponse> deleteXsdFile(@PathVariable("fileId") Long fileId) {
```

Comment:
Calling this method always throws an exception.
![image](https://user-images.githubusercontent.com/32412218/203780207-1ac7ee20-d114-412e-8c08-05d00cbfeb18.png)